IDE.sln

```diff
@@ -44,22 +44,18 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Spruce", "..\XSharp\source\
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Cosmos.Build.Tasks", "source\Cosmos.Build.Tasks\Cosmos.Build.Tasks.csproj", "{D761BA74-7CB9-4C7F-BD5F-5893A69743C9}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "IL2CPU.Debug.Symbols.Net48", "..\IL2CPU\source\IL2CPU.Debug.Symbols.Net48\IL2CPU.Debug.Symbols.Net48.csproj", "{AC3FEEDE-DA07-40BF-A1EA-56716EBC2818}"
+EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
 		Debug|Any CPU = Debug|Any CPU
 		Debug|x86 = Debug|x86
 		Release|Any CPU = Release|Any CPU
 		Release|x86 = Release|x86
+		TEST|Any CPU = TEST|Any CPU
```

Comment:
Is there a reason for a new configuration?

Reply:
Visual Studio did it by itself

Reply:
I don't think we need it as only debug is supported right now anyway

Reply:
What's the problem merging this?

Reply:
Will the extra Configuration end up on the user's vs? If yes I think it's better to remove it before merging

Reply:
If think the Test configuration is needed for unit tests